examples/demo-project/README.md

```diff
@@ -0,0 +1,43 @@
+# Example project using the MarkLogic Connector for Mulesoft
+
+This example Anypoint project demonstrates the use of the MarkLogic Connector for Mulesoft [reusable extension](https://docs.mulesoft.com/connectors/)
+for integrating a Mule application with MarkLogic. It is assumed that you have MarkLogic running on your localhost with the all of the out-of-the-box defaults available.
+
+## Example Flows
+The example has two flows.
+
+The first flow reads files from your local filesystem and inserts them into the "Documents" database in
+MarkLogic. The flow uses a Batch Job to make the flow asynchronous and to send the files to MarkLogic in
+batches of 5 in order to be efficient. It includes the "from-disk" collection on each of the documents.
+
+The second queries the "Documents" database for the documents written in the first flow. It does this by
+specifying the "from-disk" collection in an otherwise empty query. Once those documents have been retrieved,
+a Batch Job writes each document into the target directory.
+
+## Running the example
+In order to run this demonstration locally, you need to make a couple of minor changes to point the file
```

Comment:
I think the Mule-ish way to do this is to use a properties file and then have the file operations reference a property. The user can then just update the properties file. 

This example project for the 1.x connector shows an example of how to reference a properties file - https://github.com/marklogic/marklogic-mule-connector/blob/master/examples/importDocs/batch/project-mysql-importDocs-batch-flow.xml#L15 .